test/etaoin/api_test.clj

```diff
@@ -637,6 +637,16 @@
         (is (not= width width'))
         (is (not= height height'))))))
 
+(deftest test-new-window
+  (is (= 1 (count (e/get-window-handles *driver*))))
+  (let [initial-window (e/get-window-handle *driver*)
+        new-windows (for [_ (range 3)]
+                      (-> (e/new-window *driver* :tab)
```

Comment:
I'm guessing you aren't also checking `:window` (and return `:type`) because browser support varies?

Reply:
Coolio, tx for explaining!